src/wwwroot/env/QA.js

```diff
@@ -1,6 +1,6 @@
 ﻿angular.module("dopplerRelay").constant("RELAY_CONFIG", {
   baseUrl: "https://dopplerrelayqa.makingsense.com:4443",
-  cuitServiceBaseUrl: "https://apisqa.fromdoppler.com/cuit",
+  cuitServiceBaseUrl: "https://apisqa.fromdoppler.net/cuit",
```

Comment:
👍 

Did you verify this value in other environments like INT?

Reply:
Yes, they have the addresses of the cuit service with the .com and other and another completely different, do I change them?:
* DEV: cuitServiceBaseUrl: 'https://cuitserviceint.fromdoppler.net',
* INT: cuitServiceBaseUrl: "https://apisint.fromdoppler.com/cuit",

Reply:
I think so, but I am not in context, so, please confirm it by yourself before continuing.

Reply:
Url for cuit service for DEV and INT enviroments were changed to "https://apisint.fromdoppler.com/cuit", and QA to "https://apisqa.fromdoppler.com/cuit", due to invalid urls.

Reply:
👍 

You did it well, but your comment was wrong:

>Url for cuit service for DEV and INT enviroments were changed to ~"https://apisint.fromdoppler.com/cuit"~ "https://apisint.fromdoppler.net/cuit", and QA to ~"https://apisqa.fromdoppler.com/cuit"~ "https://apisqa.fromdoppler.net/cuit", due to invalid urls.